extensions/oplogPopulator/OplogPopulatorMetrics.js

```diff
@@ -132,6 +132,22 @@ class OplogPopulatorMetrics {
         }
     }
 
+    /**
+     * updates s3_oplog_populator_connectors metric
+     * when a connector is destroyed
+     * @returns {undefined}
+     */
+    onConnectorDestroyed() {
+        try {
+            this.connectors.dec({ isOld: false }, 1);
```

Comment:
this is not correct: we may remove either old or new connector : the "old" param is based on how the connector was created  (retrieved from kafka-connect or explcitely recreated), so must be tracked throughout...